src/object/srv_obj.c

```diff
@@ -3782,7 +3784,9 @@ ds_cpd_handle_one(crt_rpc_t *rpc, struct daos_cpd_sub_head *dcsh,
 			goto out;
 
 		biods[i] = vos_ioh2desc(iohs[i]);
-		rc = bio_iod_prep(biods[i], BIO_CHK_TYPE_IO);
+		rc = bio_iod_prep(biods[i], BIO_CHK_TYPE_IO,
+				  dcu->dcu_flags & ORF_CPD_BULK ?
+				  	rpc->cr_ctx : NULL, CRT_BULK_RW);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs